src/commons/utils/corio_utils.py

```diff
@@ -265,6 +275,39 @@ def setup_environment():
     os.makedirs(DATA_DIR_PATH, exist_ok=True)
 
 
+async def activate_degraded_mode() -> None:
```

Comment:
Can we have this function parameterized for no of pods failures .
Also type of pod failure(data/server/ha etc)

Reply:
No. of pods to be degraded can be taken from --degraded_pods.
parser.add_argument("-dp", "--degraded_pods", type=str, default=0,help="Number of Pods or logical nodes need tp be degraded")
Changes done accordingly.

As of now as we are making data pods debrgded for our testing hence add support for that only, if needed later we can make it for type of pod failure(data/server/ha etc..)